src/Tests/Elastic.Domain.Tests/Elasticsearch/Models/Tasks/Uninstall/UninstallDeleteDirectoriesTaskTests.cs

```diff
@@ -23,11 +24,6 @@ public class UninstallDeleteDirectoriesTaskTests : InstallationModelTestBase
 			(m, t) =>
 			{
 				var fs = t.FileSystem;
-				var session = m.Session as NoopSession;
-				fs.Directory.Exists(m.LocationsModel.DataDirectory).Should()
-					.BeFalse("{0} {1}", m.LocationsModel.DataDirectory, session);
-				fs.Directory.Exists(m.LocationsModel.ConfigDirectory).Should().BeFalse("{0}", m.LocationsModel.ConfigDirectory);
-				fs.Directory.Exists(m.LocationsModel.LogsDirectory).Should().BeFalse("{0}", m.LocationsModel.LogsDirectory);
```

Comment:
Should we not assert the opposite now rather than removing these assertations?

Reply:
Agreed 👍